src/Umbraco.Web.UI/umbraco/config/lang/ru.xml

```diff
@@ -189,7 +192,7 @@
     <key alias="memberrole">Роль участника</key>
     <key alias="membertype">Тип участника</key>
     <key alias="noDate">Дата не указана</key>
-    <key alias="nodeName">Заголовок страницы</key>
+    <key alias="nodeName">Заголовок ссылки</key>

```

Comment:
Hi @abryukhov,

Thanks for the PR! .. Almost merging it in - just wanted to be sure about this one.

I'm no Russian expert but this one seems to translate to _Headline links_ compared to the English text which is _Link title_. This field should indicate the Name/title of the link being inserted. Can you confirm that the translation works for this (I trust you know the language and variations better than Google Translate) 😄 

Best,
Claus